packaging/win32/installer.nsi

```diff
@@ -234,7 +232,7 @@ SectionEnd
 !macro InstallWinFSP
     SetOutPath "$TEMP"
     File ${WINFSP_INSTALLER}
-    ExecWait "msiexec /i ${WINFSP_INSTALLER}"
+    ExecWait "msiexec /i ${WINFSP_INSTALLER} /qn INSTALLLEVEL=1000"
```

Comment:
![image](https://user-images.githubusercontent.com/3187637/78680300-9c1cc400-78eb-11ea-89be-0a1994423077.png)

(a comment would be welcome to understand this option...)

Reply:
Not really sure how it works, but that's how you tell msiexec to not stop when a user input is required but instead go through the whole process. I think I stole that from winfuse CI.

Reply:
guess it's still better than use uuid4 as name for your options
> to enable the verbose mode, just pass the `--6b76bea6d3334e1da24e12dcc0336c1b` option (this of course disable the `--bd2e6c050730442393343403d02a2705` quiet mode)